File: Python3/FizzBuzz.py
#FizzBuzz coding interview question in Python3
#by Freya-The-Goddess

fizz = 3
buzz = 5
endNumber = 100

for i in range(1, endNumber+1):
    output = ""
    output += "Fizz" if i % fizz == 0 else ""
    output += "Buzz" if i % buzz == 0 else ""
    print(output or i)

File: Ruby/FizzBuzz.rb
#FizzBuzz coding interview question in Ruby
#by Freya-The-Goddess

fizz = 3
buzz = 5
endNumber = 100

for i in 1..endNumber do
    output = ""
    output += "Fizz" if i % fizz == 0
    output += "Buzz" if i % buzz == 0
    output = i.to_s if output.empty?
    puts output
end
